src/main/java/uk/gov/hmcts/reform/judicialapi/controller/request/UserSearchRequest.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.hmcts.reform.judicialapi.controller.request;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
+
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+public class UserSearchRequest {
+
+    @JsonProperty("searchString")
+    @NotNull(message = "cannot be null")
+    @NotEmpty(message = "cannot be empty")
```

Comment:
I think @NotEmpty will check null also

Reply:
No, took the below statement from web
`the @NotNull constraint won't allow null values for the constrained field(s). However, the field(s) can be empty.`

Reply:
since you are checking both null and empty , @NotEmpty will check both internally so no need to check both separately

Reply:
yeah, will remove @NotNull - thanks